pkg/resources/helpers.go

```diff
@@ -330,3 +331,108 @@ func ListDiff[T comparable](beforeList []T, afterList []T) (added []T, removed [
 
 	return added, removed
 }
+
+// Returns a copy of the given storage location with a 'temp_' prefix on the Name field
```

Comment:
Please update this comment (after renaming this function):
`// copySentinelStorageLocation returns a copy of storage location with a set name`